desktop/libs/notebook/src/notebook/connectors/trino.py

```diff
@@ -319,6 +319,28 @@ def _get_columns(self, database, table):
     ]
 
 
+  @query_error_handler
+  def explain(self, notebook, snippet):
+    statement = snippet['statement'].rstrip(';')
+    snippet['statement'] = 'EXPLAIN ' + statement
+    explanation = ''
+
+    if statement:
+      query_client = TrinoQuery(self.trino_request, 'USE ' + snippet['database'])
+      query_client.execute()
+      query_client = TrinoQuery(self.trino_request, 'EXPLAIN ' + statement)
+      result = query_client.execute()
+      for row in result.rows:
+        for _row in row:
+          explanation += _row
+
+    return {
+      'status': 0,
```

Comment:
Is the UI depending on this status field?

Reply:
Yes

Reply:
Will the TrinoQuery.execute() throws any exception?
If any exception, should the status be updated to -1?
Take a look existing beeswax API examples: `response = {'status': -1, 'message': ''}` are defined at the beginning of menthod.